src/sentry/taskworker/worker.py

```diff
@@ -126,26 +126,27 @@ def handle_alarm(signum: int, frame: FrameType | None) -> None:
             processed_tasks.put(
                 ProcessingResult(task_id=current.id, status=TASK_ACTIVATION_STATUS_FAILURE)
             )
-            with sentry_sdk.isolation_scope() as scope:
-                scope.fingerprint = [
-                    "taskworker.processing_deadline_exceeded",
-                    current.namespace,
-                    current.taskname,
-                ]
-                err = ProcessingDeadlineExceeded(
-                    f"execution deadline of {current.processing_deadline_duration} seconds exceeded"
-                )
-                if frame:
-                    trace = TracebackType(None, frame, frame.f_lasti, frame.f_lineno)
-                    while frame.f_back:
-                        trace = TracebackType(
-                            trace, frame.f_back, frame.f_back.f_lasti, frame.f_back.f_lineno
-                        )
-                        frame = frame.f_back
-                    err.with_traceback(trace)
-
-                sentry_sdk.capture_exception(err)
-                sentry_sdk.flush()
+            # TODO - Uncomment this once the sentry_sdk issue is resolved
+            # with sentry_sdk.isolation_scope() as scope:
+            #     scope.fingerprint = [
+            #         "taskworker.processing_deadline_exceeded",
+            #         current.namespace,
+            #         current.taskname,
+            #     ]
+            #     err = ProcessingDeadlineExceeded(
+            #         f"execution deadline of {current.processing_deadline_duration} seconds exceeded"
+            #     )
+            #     if frame:
+            #         trace = TracebackType(None, frame, frame.f_lasti, frame.f_lineno)
+            #         while frame.f_back:
+            #             trace = TracebackType(
+            #                 trace, frame.f_back, frame.f_back.f_lasti, frame.f_back.f_lineno
+            #             )
+            #             frame = frame.f_back
+            #         err.with_traceback(trace)
+
+            #     sentry_sdk.capture_exception(err)
+            #     sentry_sdk.flush()
```

Comment:
Could we log here until we figure out how to capture errors for processing deadlines.